static/app/components/reprocessedBox.tsx

```diff
@@ -18,29 +18,25 @@ type Props = {
   className?: string;
 };
 
-type State = {
-  isBannerHidden: boolean;
-};
-
-class ReprocessedBox extends Component<Props, State> {
-  state: State = {
-    isBannerHidden: localStorage.getItem(this.getBannerUniqueId()) === 'true',
-  };
+function ReprocessedBox(props: Props) {
+  const {orgSlug, reprocessActivity, groupCount, className, groupId} = props;
```

Comment:
nit if we're not using props anywhere else
```suggestion
function ReprocessedBox({orgSlug, reprocessActivity, groupCount, className, groupId} : Props) {
```